simplified-main/src/main/java/org/nypl/simplified/main/MainServices.kt

```diff
@@ -575,7 +575,7 @@ internal object MainServices {
     badgeLookup: BookCoverBadgeLookupType
   ): BookCoverProviderType {
     val execCovers =
-      NamedThreadPools.namedThreadPool(1, "cover", 19)
+      NamedThreadPools.namedThreadPool(2, "cover", 19)
```

Comment:
This makes everything feel faster since the images are loading less sequentially.

I think 4 is the default but thought 2 was a sensible place to start.